### PR TITLE
Fix & extend pkg install check

### PR DESCRIPTION
**Pull request by Borda**
# Before submitting

- [ ] Was this discussed/approved via a Github issue? (no need for typos and docs improvements)
- [x] Did you read the [contributor guideline](https://github.com/Lightning-AI/lightning/blob/master/.github/CONTRIBUTING.md), Pull Request section?
- [ ] Did you make sure to update the docs?
- [ ] Did you write any new necessary tests?

## What does this PR do?

turned out that this package may have have missing dependency `packaging`

## PR review

Anyone in the community is free to review the PR once the tests have passed.
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?

Make sure you had fun coding 🙃

### CODE REVIEW

**Review by akihironitta — APPROVED** [review]

File: .github/workflows/check-package.yml [hunk 1]
```diff
@@ -3,15 +3,28 @@ name: Check package flow
 on:
```
Comment (awaelchli) [inline .github/workflows/check-package.yml:3]: I did not review this

File: src/lightning_utilities/__init__.py [hunk 2]
```diff
@@ -3,6 +3,19 @@
 import os
 
 from lightning_utilities.__about__ import *  # noqa: F401, F403
+from lightning_utilities.core.enums import StrEnum
```
Comment (awaelchli) [inline src/lightning_utilities/__init__.py:6]: I reviewed this

**Review by awaelchli — APPROVED** [review]

File: .github/workflows/check-package.yml (hunk 1 above)
Comment (Borda) [reply .github/workflows/check-package.yml:3]: that is fine it was check by @akihironitta and all need to pass :)

File: src/lightning_utilities/__init__.py (hunk 2 above)
Comment (Borda) [reply src/lightning_utilities/__init__.py:6]: thank you :rabbit:

**Pull request merged**